src/main/java/uk/gov/hmcts/reform/civil/documentmanagement/model/DocumentType.java

```diff
@@ -47,6 +47,7 @@ public enum DocumentType {
     DEFAULT_JUDGMENT_DEFENDANT2,
     RECORD_JUDGMENT_DETERMINATION_OF_MEANS_LETTER,
     DEFAULT_JUDGMENT_COVER_LETTER,
+    JUDGMENT_BY_ADMISSION_LETTER,
```

Comment:
Is this name is too generic? as you are only doing PIN and POSt letter. Later there will be cover letter I think also.

Reply:
Agree.Renamed the name to be more specific.